### PR TITLE
Refactor RDS Cluster ResourceType

### PR DESCRIPTION
**Pull request by hongil0316**
## Description

Tackling issue: https://github.com/gruntwork-io/cloud-nuke/issues/494

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [x] Update the docs.
- [x] Run the relevant tests successfully, including pre-commit checks.
- [x] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [x] Include release notes. If this PR is backward incompatible, include a migration guide.
- [x] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

Updated [Refactor RDS Cluster ResourceType]

### CODE REVIEW

File: aws/rds_cluster.go [hunk 1]
```diff
@@ -70,7 +69,8 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
 			SkipFinalSnapshot:   awsgo.Bool(true),
 		}
 
-		_, err := svc.DeleteDBCluster(params)
+		print("deleting1")
```
Comment (arsci) [inline aws/rds_cluster.go:72]: Leftover from testing? If not we may want to make this a more detailed logging statement.

File: aws/rds_cluster.go [hunk 2]
```diff
@@ -85,27 +85,31 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
...
-				"region": *session.Config.Region,
+				"region": instance.Region,
 			})
 		} else {
 			deletedNames = append(deletedNames, name)
 			logging.Logger.Debugf("Deleted RDS DB Cluster: %s", awsgo.StringValue(name))
 		}
 	}
 
+	print("deleting2")
```
Comment (arsci) [inline aws/rds_cluster.go:96]: Same here?

File: aws/rds_cluster.go [hunk 3]
```diff
@@ -85,27 +85,31 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
...
 	if len(deletedNames) > 0 {
 		for _, name := range deletedNames {
 
-			err := waitUntilRdsClusterDeleted(svc, &rds.DescribeDBClustersInput{
+			print("deleting3")
```
Comment (arsci) [inline aws/rds_cluster.go:100]: Another

File: aws/rds_cluster.go [hunk 4]
```diff
@@ -85,27 +85,31 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
...
 			if err != nil {
 				logging.Logger.Errorf("[Failed] %s", err)
 				return errors.WithStackTrace(err)
 			}
 		}
 	}
-
-	logging.Logger.Debugf("[OK] %d RDS DB Cluster(s) nuked in %s", len(deletedNames), *session.Config.Region)
+	//
+	//print("deleting5")
```
Comment (arsci) [inline aws/rds_cluster.go:112]: One more

File: aws/rds_cluster.go [hunk 5]
```diff
@@ -85,27 +85,31 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
...
+	//logging.Logger.Debugf("[OK] %d RDS DB Cluster(s) nuked in %s", len(deletedNames), instance.Region)
```
Comment (arsci) [inline aws/rds_cluster.go:113]: Should this be uncommented?

**Review by MoonMoon1919 — APPROVED** [review]

**Review by MoonMoon1919 — APPROVED** [review]

**Review by ellisonc — APPROVED** [review]

**Pull request merged**